tests/acceptance/pageObjects/phoenixPage.js

```diff
@@ -47,6 +47,10 @@ module.exports = {
         .click('@notificationBell')
         .waitForElementVisible('@markNotificationAsReadLink')
         .click('@markNotificationAsReadLink')
+    },
+    closeAlert: function () {
+      return this.waitForElementVisible('@alertCloseIcon')
+        .click('@alertCloseIcon')
```

Comment:
after clicking you should wait till its not visible any-more

Reply:
Isn't it then causing the error I got now in drone - `Error while running "waitForElementNotVisible" command: Timed out while waiting for element ".oc-alert-close-icon" with "css selector" to be present for 10000 milliseconds`? Since it gets completely deleted from the webUI it can't be then found with the selector.

Reply:
Ah I see... I should have use present and not visible 🤦‍♂